scripts/aws/cfn/microservice.yml

```diff
@@ -690,75 +690,129 @@ Resources:
           QUEUE_URL: !Ref DeliveryPushInputsQueueURL
       Code:
         ZipFile: |
-              const AWS = require('aws-sdk')
-              const sqs = new AWS.SQS();
+              const { SQSClient, SendMessageBatchCommand } = require("@aws-sdk/client-sqs");
+              const endpointUrl = (process.env.ENV=='LOCAL'?'http://localstack:4566':null );
+              const sqs = new SQSClient({ region: process.env.REGION, endpoint: endpointUrl});
```

Comment:
@fabriziolana io farei così: crei un oggetto config che passi in input a SQSClient e gli imposti la property `endpoint` solo se sei in **LOCAL**

Reply:
ok rivista assegnazione params